### PR TITLE
Bug/bi 1158

### PR DESCRIPTION
**Pull request by davedrp**
see https://breedinginsight.atlassian.net/browse/BI-1158

### CODE REVIEW

File: src/components/trait/TraitDetailPanel.vue [hunk 1]
```diff
@@ -303,9 +303,7 @@
 
     abbreviationsSynonymsString(synonymsMaxLength: number) : string | undefined {
       let abbSyn = "";
-      if (this.data && this.data.abbreviations && this.data.abbreviations.length > 0) {
-        abbSyn = this.data.abbreviations[0];
-      }
+
       if (this.data && this.data.synonyms && this.data.synonyms.length > 0) {
         // Up to synonymsMaxLength synonyms will be shown before , ... cutoff
         const synonyms = this.data.synonyms.slice(0, Math.min(this.data.synonyms.length, synonymsMaxLength)).join(", ");
```
Comment (ctucker3) [inline src/components/trait/TraitDetailPanel.vue:309]: Are all of these abbreviation changes intended for this card? The card you linked was only for the changed file template.

**Review by ctucker3 — CHANGES_REQUESTED** [review]

**Comment by ctucker3** [comment]
What branch of bi-api should I be running this on? When running on develop with the file below I get the error in the image:
[TraitsTest_BI1148.xls](https://github.com/Breeding-Insight/bi-web/files/7590280/TraitsTest_BI1148.xls)

BI1158_failed_import

**Comment by davedrp** [comment]
> What branch of bi-api should I be running this on? When running on develop with the file below I get the error in the image: [TraitsTest_BI1148.xls](https://github.com/Breeding-Insight/bi-web/files/7590280/TraitsTest_BI1148.xls)
> 
> BI1158_failed_import

**There is a bug/bi-1158 branch in bi-api**

File: src/components/trait/TraitDetailPanel.vue (hunk 1 above)
Comment (davedrp) [reply src/components/trait/TraitDetailPanel.vue:309]: After a conversation with Tim, I removed all references to the abbreviation data element.  (I have now added a brief [comment](https://breedinginsight.atlassian.net/browse/BI-1158?focusedCommentId=12167) to the Jira story )

**Review by ctucker3 — APPROVED** [review]
Ran it with the bi-api BI-1158 branch and it worked as expected.

File: src/breeding-insight/model/Trait.ts [hunk 2]
```diff
@@ -29,7 +29,7 @@ export class Trait {
   traitDescription?: string;
   method?: Method;
   scale?: Scale;
-  abbreviations?: Array<string>;
+  //abbreviations?: Array<string>;
```
Comment (dmeidlin) [inline src/breeding-insight/model/Trait.ts:32]: Looks like a line that was commented out initially and should now be deleted.

Comment (davedrp) [reply src/breeding-insight/model/Trait.ts]: Deleted

**Review by dmeidlin — APPROVED** [review]

**Pull request merged**